File: app/src/main/java/com/worldventures/dreamtrips/social/ui/bucketlist/presenter/ForeignBucketItemDetailsPresenter.java
package com.worldventures.dreamtrips.social.ui.bucketlist.presenter;

import com.worldventures.dreamtrips.social.ui.bucketlist.model.BucketItem;

public class ForeignBucketItemDetailsPresenter extends BucketItemDetailsPresenter {

   public ForeignBucketItemDetailsPresenter(BucketItem.BucketType type, BucketItem bucketItem, int ownerId) {
      super(type, bucketItem, ownerId);
   }
}

File: app/src/main/java/com/worldventures/dreamtrips/social/ui/bucketlist/presenter/ForeignBucketItemDetailsPresenter.kt
package com.worldventures.dreamtrips.social.ui.bucketlist.presenter

import com.worldventures.dreamtrips.social.ui.bucketlist.model.BucketItem

class ForeignBucketItemDetailsPresenter(type: BucketItem.BucketType, bucketItem: BucketItem, ownerId: Int) :
      BucketItemDetailsPresenter(type, bucketItem, ownerId)
